cpp/src/groupby/hash/compute_mapping_indices.cuh

```diff
@@ -71,7 +76,7 @@ __device__ void find_global_mapping(cooperative_groups::thread_block const& bloc
     auto const input_idx = shared_set_indices[idx];
     auto const key_idx   = *global_set.insert_and_find(input_idx).first;
 
-    global_mapping_indices[block.group_index().x * GROUPBY_SHM_MAX_ELEMENTS + idx] = key_idx;
+    global_mapping_indices[block.group_index().x * GROUPBY_CARDINALITY_THRESHOLD + idx] = key_idx;
```

Comment:
Is `GROUPBY_SHM_MAX_ELEMENTS` still relevant or it can be depreacted also?
https://github.com/rapidsai/cudf/blob/51b109136f26638e0596a95cf693e3cbe0f2f531/cpp/src/groupby/hash/helpers.cuh#L34-L38

Reply:
Yes it is still relevant, since it is used for allocating the shared mem buffer of the block-wise set. We may insert up to 256 elements into the shared set, but we will fallback if there are more than 128.